lib/OpenQA/Scheduler/Client.pm

```diff
@@ -22,6 +22,25 @@ use OpenQA::Utils 'service_port';
 has client => sub { OpenQA::Client->new(api => 'localhost') };
 has port   => sub { service_port('scheduler') };
 
+sub embed_server_for_testing {
```

Comment:
Shouldn't this actually be in the test module? :thinking:

Reply:
I implemented it in consistency with `embed_server_for_testing` in the other clients. But it would make sense to unify these functions and move them to the test utilities. @kraih, right?

Reply:
I moved it to the test utilities and merged it with the other `embed_server_for_testing` function.